tools/rbs-bearer-token.sh

```diff
@@ -0,0 +1,55 @@
+# Use this script to get the RBS Bearer token to use to auth w/ RBS swagger page
+# Swagger page allows you to check how many projects are available for the pool
```

Comment:
```suggestion
#!/bin/sh
set -eu

# Use this script to get the RBS Bearer token to use to auth w/ RBS swagger page
# Swagger page allows you to check how many projects are available for the pool
```

Reply:
I also recommend running `shellcheck` on your script.

Reply:
Ok - I ran shellcheck. Everything passes except for a warning message about the select loop `SC3008 (warning): In POSIX sh, select loops are undefined.` Given it's just a warning, I would like to keep the select loop.

Reply:
If there's no need for this script to be POSIX compatible, you can just switch to `#!/bin/bash` and then `shellcheck` should stop complaining.

Reply:
That worked!